### DISCUSSION TITLE
Support for Bootstrap 5 Floating Labels

### DISCUSSION THREAD

**Discussion by lupo112** [orchidjs/tom-select#149] [Ideas]
Can you add support for bootstrap 5 floating labels?

https://getbootstrap.com/docs/5.0/forms/floating-labels/

Currently, it is not styled for this type of layout and is showing as follows:
![image](https://user-images.githubusercontent.com/1206903/126805882-c68cf6c0-b62b-4ee9-b42b-6d7d5b57ee31.png)

The last input is regular text input which shows as expected, previous 3 selects are without border and with border in the content of .form-floating.

**Comment by mvxproject** [comment]
Related to : https://github.com/orchidjs/tom-select/discussions/337
